doc/releases/mimic.rst

```diff
@@ -69,8 +69,39 @@ Notes
 Instructions
 ~~~~~~~~~~~~
 
+#. If your cluster was originally installed with a version prior to
+   Luminous, ensure that it has completed at least one full scrub of
+   all PGs while running Luminous.  Failure to do so will cause your
+   monitor daemons to refuse to join the quorum on start, leaving them
+   non-functional.
+
+   If you are unsure whether or not your Luminous cluster has
+   completed a full scrub of all PGs, you can check your cluster's
+   state with the following steps:
+
+   * Retrieve your OSD map: ``ceph osd getmap -o  /tmp/osdmap``
+   * Examine it: ``osdmaptool --print /tmp/osdmap``
+   * Check the ``flags`` entry. In order to be able to proceed to Mimic,
+     your OSD map must include the ``recovery_deletes`` and
+     ``purged_snapdirs`` flags.
+
+   If your OSD map does not contain either of these flags, you can
+   simply wait for approximately 24-48 hours, which in a standard
+   cluster configuration should be ample time for all your placement
+   groups to be scrubbed at least once, and then repeat the above
+   process to recheck.
```

Comment:
I don't think this is accurate in general — some people have trouble getting a full scrub every *week*.

Reply:
Note that it's a shallow scrub.. those are comparatively quick. I think it's the deep scrubs that users struggle with